<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionControl_Dialog"/>
    <addaction name="actionMCL_Dialog"/>
    <addaction name="actionSensor_View"/>
    <addaction name="actionDistance_Matrix"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionControl_Dialog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Control Dialog</string>
   </property>
  </action>
  <action name="actionMCL_Dialog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MCL Dialog</string>
   </property>
  </action>
  <action name="actionSensor_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sensor View</string>
   </property>
  </action>
  <action name="actionDistance_Matrix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Distance Matrix</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
